week-1/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,9 @@
 // Write your function here
-
-var greeting = createGreeting("Daniel");
+function createGreeting(name, age) {
+  return "hello,my name is " + name;
+  "and i am old" + age;
```

Comment:
This is a good solution to the exercise. But I see you also wanted to log the age. To do this, instead of the ";" after name you need a "+" to concatenate the strings together.